lib/internal/fs/recursive_watch.js

```diff
@@ -202,6 +202,9 @@ class FSWatcher extends EventEmitter {
         this.emit('change', 'rename', pathRelative(this.#rootPath, file));
       } else if (currentStats.isDirectory()) {
         this.#watchFolder(file);
+      } else {
```

Comment:
We need to check if the file is actually changed through `file.mtimeMs`.

Reply:
this code is triggered only when the file is changed